bibliopixel/led/cube.py

```diff
@@ -0,0 +1,93 @@
+import math, threading, time
```

Comment:
Cube is used by my testing for the new visualizer. I will however get rid of the commented stuff.
And your new stuff has documentation??